config/samples/avroschema.json

```diff
@@ -14,6 +14,10 @@
       "name": "account",
       "type": "string"
     },
+    {
```

Comment:
not a blocker but fyi this is part of the future xjoin implementation so it's not required here.